string name
geometry_msgs/PoseWithCovarianceStamped pose_cov_stamped
float32 radius
